/motor_calibration:
  ros__parameters:
    port: /dev/ttyTHS1
    username: eduardohufg
/motor_calibration_cli:
  ros__parameters:
    port: /dev/ttyTHS1
    username: eduardohufg
/motor_gen_control:
  ros__parameters:
    kd: 0.005
    ki: 0.11
    kp: 0.11
    max_angle: 180.0
    min_angle: -10.0
    port: /dev/ttyTHS1
    zero_encoder_pos: 2011.0
